packages/infra/lib/ihe-stack.ts

```diff
@@ -36,11 +35,18 @@ export function createIHEStack(stack: Construct, props: IHEStackProps) {
     },
   });
 
+  // get the certificate form ACM
+  const certificate = cert.Certificate.fromCertificateArn(
+    stack,
+    "IHECertificate",
+    props.config.ihe.CERT_ARN
```

Comment:
If this is not required in all envs, we should check it like we do w/ the URL on like 26. Disregard otherwise.